README.md

```diff
@@ -46,31 +42,12 @@ During project where used following libraries:
 ## Installation
 
 Project requires [Pytorch](https://pytorch.org/)  1.11.0.
-
 Install the dependencies via pip or conda install managers.
+requirements.txt
```

Comment:
The environment file is required. You can use whatever you like

Reply:
Of course, I added yaml file. Regarding wandb's log files - since I used wandb in first time I tried it's functionalities and forgot to delete unnecessary logs (